sw/airborne/modules/e_identification_fr/e_identification_fr.c

```diff
@@ -138,8 +139,11 @@ void e_identification_fr_init()
 
 void e_identification_fr_periodic()
 {
-
-  if (gps.fix) {
+  if(autopilot_in_flight()) {
```

Comment:
`autopilot_in_flight` is currently only working on rotorcraft, but fixedwing can use `launch` variable instead (use ifdef `FIXEDWING_FIRMWARE` or `ROTORCRAFT_FIRMWARE`)
see #2641